packages/jaspr/pubspec.yaml

```diff
@@ -5,7 +5,7 @@ homepage: https://github.com/schultek/jaspr
 issue_tracker: https://github.com/schultek/jaspr/issues
 
 environment:
-  sdk: '>=2.16.0 <3.0.0'
+  sdk: '>=2.17.0 <3.0.0'
```

Comment:
I updated this on the main branch so you can merge this into yours.

Reply:
Yes I know I updated it already but now I have this problem: https://github.com/schultek/jaspr/issues/20
Can you fix it please?